marklogic-data-hub-central/ui/src/components/base-entities-facet/base-entities-facet.tsx

```diff
@@ -150,7 +150,7 @@ const BaseEntitiesFacet: React.FC<Props> = (props) => {
               <div
                 key={name}
                 aria-label={`base-entities-${name}`}
-                style={{backgroundColor: finalColor}}
+                style={{backgroundColor: finalColor, borderStyle: "solid", borderWidth: "1px", borderColor: "#d9d9d9", borderRadius: "4px"}}
```

Comment:
@briantang can we move this styles to a scss file ?

Reply:
@iveyMarklogic Because the background color needed to be set by a variable, I had to keep the inline-styling. Do you suggest I try to use both the stylesheet and inline?